docs-doctor/event.json

```diff
@@ -0,0 +1,3 @@
+{
+  "number": 1
+}
```

Comment:
What is this file for?

Reply:
This file is used by GitHub local-action to specify the GitHub Action Event.